package.json

```diff
@@ -17,7 +17,8 @@
     "prepare": "husky install",
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
-    "cypress:headed": "cypress run --headed"
+    "cypress:headed": "cypress run --headed",
+    "update-types": "dotenv -e .env -- sh -c 'npx supabase gen types --lang=typescript --project-id $PROJECT_ID > supabase/types.ts'"
```

Comment:
`sh` is also not cross platform. Please have a look at https://www.npmjs.com/package/cross-env

Reply:
From what I see here,`cross-env` is good for setting environment variables in a cross-platform way but does not handle `.env` files.

Reply:
Correct which is why you should combine `cross-env-shell` to run the script and `dotenv` to load the environment file.